pkg/terraform/install.go

```diff
@@ -28,8 +30,25 @@ func (m *Mixin) Install() error {
 		step.Flags = append(step.Flags, builder.NewFlag("input=false"))
 	}
 
-	for _, k := range sortKeys(step.Vars) {
-		step.Flags = append(step.Flags, builder.NewFlag("var", fmt.Sprintf("'%s=%s'", k, step.Vars[k])))
+	if step.CreateVarFile {
+		vf, err := m.FileSystem.Fs.Create(path.Join(m.WorkingDir, m.TerraformVarsFilename))
+		if err != nil {
+			return err
+		}
+		defer vf.Close()
+		vbs, err := json.Marshal(step.Vars)
+		if err != nil {
+			return err
+		}
+		_, err = vf.Write(vbs)
+		if err != nil {
+			return err
+		}
+		fmt.Printf("TF var file created:\n%s\n", string(vbs))
+	} else {
+		for _, k := range sortKeys(step.Vars) {
+			step.Flags = append(step.Flags, builder.NewFlag("var", fmt.Sprintf("'%s=%s'", k, step.Vars[k])))
+		}
```

Comment:
Another note (not specifically about this line): [Upgrade](https://github.com/getporter/terraform-mixin/blob/main/pkg/terraform/upgrade.go#L5) still uses this `m.Install()` method for its logic.  I'd say, assuming we go with the install-specific logic added here, we update Upgrade to implement its own logic (basically, what `m.Install()` was prior to this PR).